scripts/uninstall.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Remove munkireportinfo script
+rm -f "${MUNKIPATH}preflight.d/munkireportinfo"
```

Comment:
I don't think this file ever existed :-D

Reply:
I believe you're correct. Wouldn't we want to remove it if it exists in the future?